platforms/core-configuration/declarative-dsl-core/src/main/resources/org/gradle/internal/declarativedsl/parsing/grammar_test_input.txt

```diff
@@ -15,4 +14,10 @@ f(x=y)
 f.g.h.i.j.k(test)
 f(1, x, "s", g())
 f(a = b, c = d)
-f(1).g(2).h(3)
+
+// errors
+// a.b.c = 1
+// d = a.b.c
+// a.b {d = 1}
+// a.b.c {d = 1}
+// f(1).g(2).h(3)  // TODO: do we want to allow for this?
```

Comment:
Not in the "minimal" grammar for our currently supported set of features. We will want to keep it in the grammar that describes the unsupported Kotlin features as well.